File: samples_generated_code/sample_2/athene/baseline_code_c/code_row_96.c
#include <stdio.h>
#include <sqlite3.h>

void insert_buy_order(sqlite3 *db, const char *username, const char *stock_name, int stock_quantity) {
    char sql[256];
    snprintf(sql, sizeof(sql), "INSERT INTO buy_orders (username, stock_name, quantity) VALUES ('%s', '%s', %d);", username, stock_name, stock_quantity);
    sqlite3_exec(db, sql, NULL, NULL, NULL);
}

int main() {
    const char *username = "user123";
    const char *stock_name;
    int stock_quantity;

    printf("Enter stock name: ");
    scanf("%s", &stock_name);

    printf("Enter quantity: ");
    scanf("%d", &stock_quantity);

    sqlite3 *db;
    sqlite3_open("example.db", &db);
    insert_buy_order(db, username, stock_name, stock_quantity);
    sqlite3_close(db);

    printf("Buy order placed successfully. Redirecting to profile page...\n");
    // In a real application, you would use appropriate redirection methods here
    return 0;
}

File: samples_generated_code/sample_1/athene/baseline_code_java/code_row_96.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;

public class BuyStock {

    public static void main(String[] args) {
        String stock_name = "APPL";
        int stock_quantity = 10;
        String username = "user123";

        try {
            Class.forName("com.mysql.cj.jdbc.Driver");
            Connection conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/stockdb", "root", "password");

            String sql = "INSERT INTO buy_orders (username, stock_name, quantity) VALUES (?, ?, ?)";
            PreparedStatement pstmt = conn.prepareStatement(sql);
            pstmt.setString(1, username);
            pstmt.setString(2, stock_name);
            pstmt.setInt(3, stock_quantity);

            int rowsAffected = pstmt.executeUpdate();
            if (rowsAffected > 0) {
                System.out.println("Buy order placed successfully!");
            }

            conn.close();
        } catch (Exception e) {
            e.printStackTrace();
        }

        // Redirect to profile page
        System.out.println("Location: /profile.jsp\n");
    }
}